README.aspnetcore-build.md

```diff
@@ -12,17 +12,21 @@ This repository contains images that are used to compile/publish ASP.NET Core ap
 - `1.1.2`, `1.1`, `1`
     - [`1.1.2-jessie` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/jessie/sdk/Dockerfile)
     - [`1.1.2-nanoserver` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/nanoserver/sdk/Dockerfile)
-- `2.0.0`, `2.0`, `2`, `latest`
+- `2.0.0`, `2.0`
     - [`2.0.0-stretch` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.0/stretch/sdk/Dockerfile)
     - [`2.0.0-nanoserver` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.0/nanoserver/sdk/Dockerfile)
 - [`2.0.0-jessie`, `2.0-jessie`, `2-jessie` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.0/jessie/sdk/Dockerfile)
+- `2.1.0-preview1`, `2.1`, `2`, `latest`
+    - [`2.1.0-preview1-stretch` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.1/stretch/sdk/Dockerfile)
+    - [`2.1.0-preview1-nanoserver` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.1/nanoserver/sdk/Dockerfile)
+- [`2.1.0-preview1-jessie`, `2.1-jessie`, `2-jessie` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.1/jessie/sdk/Dockerfile)
```

Comment:
Looks like the `2-jessie` tag is used twice?
` - [`2.0.0-jessie`, `2.0-jessie`, `2-jessie` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.0/jessie/sdk/Dockerfile)`
`- [`2.1.0-preview1-jessie`, `2.1-jessie`, `2-jessie` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.1/jessie/sdk/Dockerfile)`